addons/mail/static/src/owl/components/chatter/chatter.js

```diff
@@ -14,37 +15,31 @@ class Chatter extends Component {
      */
     constructor(...args) {
         super(...args);
-        this.state = useState({ isAttachmentBoxVisible: false });
+        this.state = useState({
+            isComposerLog: false,
+            isAttachmentBoxVisible: false,
+            isComposerVisible: false,
+        });
         this.storeDispatch = useDispatch();
         this.storeGetters = useGetters();
-        this.storeProps = useStore((state, props) => {
-            const thread = this.storeGetters.thread({
-                _model: props.model,
-                id: props.id,
-            });
+        this.storeProps = useStore((state, { chatterLocalId }) => {
+            const chatter = state.chatters[chatterLocalId];
+            const thread = state.threads[chatter.threadLocalId];
             return {
+                composerLocalId: thread ? thread.composerLocalId : undefined,
                 threadLocalId: thread ? thread.localId : undefined,
             };
         });
         this._threadRef = useRef('thread');
     }
 
-    mounted() {
-        // TODO {xdu}
-        // Need to say to the underlying Thread that there is no long polling
-        // So when doing message_post, you need to do a read on the record and
-        // then verify in the message_ids if there are missing messages or not
-        // and then make a message_format call for each message (including the
-        // one posted)
-        this.storeDispatch('initChatter', {
-            model: this.props.model,
-            id: this.props.id,
-        });
-    }
-
     //--------------------------------------------------------------------------
     // Handlers
-    //--------------------------------------------------------------------------
+    //--------------------------------------------------------------------------a
+
+    _onComposerMessagePosted() {
+        this.storeDispatch('loadNewMessagesOnThread', this.storeProps.threadLocalId);
```

Comment:
something looks off with handling postMessage:
https://github.com/odoo-dev/odoo/blob/master-mail-owl-xdu/addons/mail/static/src/owl/components/composer/composer.js#L164

Concurrent stuffs happening: `message_post` RPC + `loadNewMessagesOnThread`.
Could miss loading newly posted message.

I think the correct way to do it is to have a single store dispatch when posting a message: it makes the `message_post` RPC, auto-unlink attachments from composer, and auto-loads messages on thread if the message post happens from a record thread